bitex/exceptions.py

```diff
@@ -17,23 +17,27 @@ class IncompleteCredentialsWarning(UserWarning):
 
 
 class IncompleteCredentialConfigurationWarning(UserWarning):
-    """Raised when a required parameter for authentication isn't found in a config file."""
+    """Raised when a required parameter for authentication isn't found in a
+    config file.
+    """
 
     pass
 
 
 class IncompleteCredentialsError(Exception):
     """Raised when attempting to call private_query() of an API.
 
-    For example when missing essential values to create a message signature (key, secret
-    or additional other values, dependent on the exchange).
+    For example when missing essential values to create a message signature
```

Comment:
this is invalid - Docstrings without parameters should be a single line. (PEP257)